OsmAnd/res/values/strings.xml

```diff
@@ -12,31 +12,31 @@
-    <string name="markers_history_is_disabled_descr">Enable history to get list of visited map markers.</string>
-    <string name="history_is_disabled_descr">Enable history to get quicker access to last used results.</string>
-    <string name="is_disabled">%1$s is disabled</string>
-    <string name="clear_all_history_warning">Are you sure you want to clear all history in the app?\n\nThe search, navigation and marker history will be cleared.</string>
+    <string name="no_plugins_enabled">If you want to use any plugins there is a full list in \'Menu\' → \'Plugins\'.</string>
+    <string name="shared_storage_warning_description">If you are using OsmAnd 4.2 or later, change from shared to available storage before Nov. 2021 to retain access to offline maps, GPX tracks, etc.</string>
+    <string name="search_history_is_disabled_descr">You can turn on search history in the settings</string>
+    <string name="markers_history_is_disabled_descr">Turn on history to get a list of visited map markers.</string>
+    <string name="history_is_disabled_descr">Turn on history to access last used results quicker.</string>
+    <string name="is_disabled">%1$s is off</string>
+    <string name="clear_all_history_warning">Clear all app history including search, navigation and marker history?</string>
     <string name="clear_all_history">Clear all history</string>
     <string name="history_actions_descr">You can export history into a file.</string>
     <string name="map_markers_history">Map markers history</string>
     <string name="navigation_history">Navigation history</string>
-    <string name="history_preferences_descr">You can disable history logging individually for each history type.</string>
+    <string name="history_preferences_descr">You can turn off logging for each history type.</string>
     <string name="shortcut_start_recording">Start Recording</string>
     <string name="shortcut_navigate_to">Navigate To…</string>
-    <string name="shortcut_navigate_to_work">Navigate to Work</string>
-    <string name="shortcut_navigate_to_home">Navigate to Home</string>
+    <string name="shortcut_navigate_to_work">Work</string>
+    <string name="shortcut_navigate_to_home">Home</string>
     <string name="android_auto_map_placeholder_title">The map is displayed on the vehicle screen.</string>
     <string name="android_auto_map_placeholder_descr">Disconnect your device from Android Auto to view the map in the app.</string>
     <string name="android_auto_purchase_request_title">Purchase OsmAnd Pro or Maps+ to use OsmAnd with Android Auto</string>
-    <string name="continue_on_phone">Continue on Phone</string>
-    <string name="location_access_request_title">OsmAnd needs access to location in order to navigate</string>
+    <string name="continue_on_phone">Continue on mobile device</string>
+    <string name="location_access_request_title">OsmAnd needs location access to navigate</string>
```

Comment:
Leave original.

Reply:
There is no "in order to" anywhere.
The rest is "Grant location data access." and so on.